src/Lumi/Components2/Banner.purs

```diff
@@ -0,0 +1,223 @@
+                  $ _ { content =
+                          [ case props.title of
+                              Just title ->
+                                box
+                                $ S._alignSelf S.Start
+                                $ style_
+                                    ( css
+                                      { marginBottom: int 8
+                                      }
+                                    )
+                                $ _ { content = [ title ]
+                                    }
+                              _ -> mempty
+                          , box
+                            $ S._align S.Center
+                            $ S._row
+                            $ S._flex
+                            $ _ { content =
+                                    [ case props.icon of
+                                        Just icon ->
+                                          box
+                                          $ S._alignSelf S.Start
+                                          $ S.onDesktop (S._alignSelf S.Center)
+                                          $ style_
+                                              ( css
+                                                  { marginRight: int 16
+                                                  }
+                                              )
+                                          $ _ { content = [ icon ]
+                                              }
+                                        _ -> mempty
+                                    ]
+                                    <> props.content
+                                }
+                          ]
+                      }
+                , Monoid.guard props.dismissable
+                    $ button
+                    $ S._alignSelf S.Start
+                    $ dismissButtonStyle
+                    $ _ { content =
+                            [ Icon.icon
+                                { style: R.css { fontSize: "12px" }
+                                , type_: Icon.Remove
+                                }
+                            ]
+                        , onPress = setVisible \_ -> false
+                        }
+                ]
+            }
+  where
+    defaults :: Record BannerProps
+    defaults =
+      { component: Banner
+      , title: Nothing
+      , content: []
+      , dismissable: false
+      , icon: Nothing
+      }
+
+    dismissButtonStyle :: forall props. PropsModifier props
+    dismissButtonStyle =
+      S._interactive
+      <<< style \(LumiTheme { colors }) ->
+            css
+              { ariaLabel: str "dismiss"
+              , outline: str "none"
+              , border: str "none"
+              , background: str "none"
+              , color: color colors.black1
+              , minWidth: str "initial"
+              , padding: int 0
+              , "@media (min-width: 860px)": nested $ css
```

Comment:
We should be using `desktopQuery` here so that this is affected by an eventual change of our definition of a desktop-sized screen.

Reply:
This was to override a media query style from our Components2 `Button` but now that we're using `R.button` I have removed these.